### PR TITLE
Re-export the frost-core traits and rand-core as part of top-level impls API

### LINKED ISSUE #296: Re-export frost-core, rand-core traits needed by consumers of top-level libraries

### PR DESCRIPTION
**Pull request by dconnolly**
Resolves #296

### CODE REVIEW

File: frost-ed25519/src/lib.rs [hunk 1]
```diff
@@ -11,11 +11,15 @@ use curve25519_dalek::{
 use rand_core::{CryptoRng, RngCore};
 use sha2::{Digest, Sha512};
 
-use frost_core::{frost, Ciphersuite, Field, FieldError, Group, GroupError};
+pub use frost_core::frost;
```
Comment (conradoplg) [inline frost-ed25519/src/lib.rs:14]: This uses `pub` but others don't, was this intended?

File: frost-ristretto255/src/lib.rs [hunk 2]
```diff
@@ -11,11 +11,15 @@ use curve25519_dalek::{
 use rand_core::{CryptoRng, RngCore};
 use sha2::{Digest, Sha512};
 
-use frost_core::{frost, Ciphersuite, Field, FieldError, Group, GroupError};
+pub use frost_core::frost;
```
Comment (conradoplg) [inline frost-ristretto255/src/lib.rs:14]: This uses `pub` but others don't, was this intended?

**Review by conradoplg — CHANGES_REQUESTED** [review]

File: frost-ristretto255/src/lib.rs (hunk 2 above)
Comment (dconnolly) [reply frost-ristretto255/src/lib.rs]: Whoops, fixed in cf6b639

File: frost-ed25519/src/lib.rs (hunk 1 above)
Comment (dconnolly) [reply frost-ed25519/src/lib.rs]: Whoops, fixed in cf6b639

**Review by conradoplg — APPROVED** [review]

**Pull request merged**